build.gradle.kts

```diff
@@ -4,6 +4,7 @@
  */
 import io.gitlab.arturbosch.detekt.Detekt
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
```

Comment:
⚠️ **[detekt]** <detekt.ForbiddenComment> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This comment contains 'TODO:' that has been defined as forbidden in detekt.